CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Prevent internal server error when course runs sync webhook targets
```

Comment:
IMO the commit body "one act warning and some unmocked requests" must focus more on the why and could be more self explanatory like "some tests were emitting warnings due to act misusage and unmocked requests, so we decided to fix those".